ethcore/trace/src/db.rs

```diff
@@ -332,19 +337,43 @@ impl<T> TraceDatabase for TraceDB<T> where T: DatabaseExtras {
 	}
 }
 
+/// Bridge between TraceDb and Blockchain.
+impl DatabaseExtras for BlockChain {
```

Comment:
I'm not 100% sure whether this trait impl belongs here. The whole idea was that tracedb and blockchaindb crates are independent and don't know about each other. Placing this implementation here makes blockchaindb one of the tracedb dependencies

Reply:
Great idea, fixing.